packages/near-fast-auth-signer/src/components/SignMultichain/bitcoin/schema.ts

```diff
@@ -0,0 +1,17 @@
+import * as yup from 'yup';
+
+export const bitcoinSchema = yup.object().shape({
+  derivationPath:       yup.string().required('derivationPath is required'),
+  to:                   yup.string().required('to is required'),
+  value:                yup.mixed<bigint>().required('value is required'),
+  derivedPublicKey:     yup.string().required('derivedPublicKey is required'),
```

Comment:
I think this is not necessary anymore, you can derive it from the derivationPath